apps/flows/src/app/home/home.component.html

```diff
@@ -0,0 +1,53 @@
+<main>
+  <section>
```

Comment:
Not important:

I think the containing `<section>` is redundant and should be removed.

Reply:
Done 👍